product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -610,6 +707,13 @@ def read(self, fields=None, load='_classic_read'):
                 except:
                     continue
             res[0].update(dynamic_vals)
+
+        if not self.env.context.get('from_fvg'):
+            for res_data in res:
+                if res_data.get('stored_support_vals'):
+                    res_data.update(
+                        ast.literal_eval(res_data['stored_support_vals'])
+                    )
```

Comment:
If the code that updates the support fields is in the write method do we still need this?